nexus/test-utils/src/resource_helpers.rs

```diff
@@ -1060,32 +1060,20 @@ pub async fn detach_ip_address_from_igw(
 pub async fn assert_ip_pool_utilization(
     client: &ClientTestContext,
     pool_name: &str,
-    ipv4_allocated: u32,
-    ipv4_capacity: u32,
-    ipv6_allocated: u128,
-    ipv6_capacity: u128,
+    remaining: f64,
```

Comment:
would it simplify the callers to have this be `allocated`?

Reply:
Done in 45419e0ba